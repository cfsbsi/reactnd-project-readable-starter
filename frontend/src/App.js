import React, {Component} from 'react';
import {ListGroup, ListGroupItem, Panel, Col} from 'react-bootstrap';
import {Route, Link} from 'react-router-dom';


class App extends Component {
    render() {
        return (
            <div className="App">
                <Route exact path="/" render={() => (
                    <Col md="8" mdOffset="2">
                        <Panel header="Redux">
                            <ListGroup>
                                <ListGroupItem>Link 1</ListGroupItem>
                                <ListGroupItem>Link 2</ListGroupItem>
                            </ListGroup>
                        </Panel>
                    </Col>
                )}/>
            </div>
        );
    }
}

export default App;
