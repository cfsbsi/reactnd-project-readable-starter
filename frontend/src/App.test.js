import React from 'react';
import App from './App';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';

const mockStore = configureStore();

const store = {categories: []};

it('renders without crashing', () => {
  expect(shallow(<App store={mockStore(store)}/>)).toMatchSnapshot();
});
