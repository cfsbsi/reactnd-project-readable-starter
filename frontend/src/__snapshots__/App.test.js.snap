// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
</div>
`;
